README.md

```diff
@@ -129,3 +125,96 @@ Since the feature flags are requested from LaunchDarkly after DOM load, there ma
 
 #### `renderDefaultCallback` : `function` (optional)
 This callback is provided for cases where you want to render something by default, think of it when your feature flag is "off" or falsy.
+
+---
+
+### SSR Support
+SSR is opt-in and you need to specify the initial set of feature flag keys and values through
+the `bootstrap` property on `clientOptions`:
```

Comment:
`cache` might make more sense here. Bootstrapping is more about code than data, and this is just supplying data.

Reply:
I definitely agree here, `bootstrap` was used to keep consistency with the actual LaunchDarkly client: https://github.com/launchdarkly/js-client#bootstrapping

Reply:
Not sure if you have any thoughts about deviating from that, I mean `react-launch-darkly` is mostly a wrapper around the actual LD client. So we don't necessarily have to stay with their naming conventions.

Reply:
Best to keep their naming, then.